config/health-checker-kubelet.json

```diff
@@ -25,6 +25,7 @@
         "--component=kubelet",
         "--enable-repair=true",
         "--cooldown-time=1m",
+        "--loopback-time=5m",
```

Comment:
Maybe let the default config use the default loopback time to keep the same behavior as it was before this change?

Reply:
Default loopback time is zero, so if not this set. It is the same behavior as it was before. Here is a example to ues.

Reply:
I set to 0.

Reply:
Thanks!